# qiskit.providers.ibmq.credentials.CredentialsNotFoundError



`CredentialsNotFoundError(*message)`

Errors raised when the credentials are not found.

Set the error message.



`with_traceback()`

Exception.with\_traceback(tb) â€“ set self.\_\_traceback\_\_ to tb and return self.
